### PR TITLE
docs: suggest users install package via composer only

### LINKED ISSUE #98: Suggestion: Open development branch for accepting PRs to next major version
To make larger changes I would suggest not merging such PRs directly into `main`, since you have this in the readme: 

```json
{
  "require": {
    "imgix/imgix-php": "dev-main"
  }
}
```

This could mean that people have the main branch running directly in their apps, and that this would break while merging PRs with breaking changes.

### PR DESCRIPTION
**Pull request by sherwinski**
TSIA
fixes #98

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,18 +37,6 @@ Install through composer with:
...
-    "imgix/imgix-php": "dev-main"
-  }
-}
-```
-
-And include the global `vendor/autoload.php` autoloader.
-
 ### Standalone
 
 Just copy the files to your project, and include the `src/autoload.php` file. We recommend using Composer if at all possible.
```
Comment (luqven) [inline README.md:42]: Shouldn't we remove this as well if we're removing the above?

**Review by luqven — APPROVED** [review]
LGTM just one question

File: README.md (hunk 1 above)
Comment (sherwinski) [reply README.md:42]: Definitely, good call

Comment (sherwinski) [inline README.md:42]:
```suggestion
```

**Pull request merged**